apps/stripe/src/__tests__/mocks/transaction-refund-request-event.ts

```diff
@@ -0,0 +1,21 @@
+import { TransactionRefundRequestedEventFragment } from "@/generated/graphql";
+
+import { mockedSaleorChannelId } from "./constants";
+import { mockedStripePaymentIntentId } from "./mocked-stripe-payment-intent-id";
+
+export const getMockedTransactionRefundRequestedEvent =
+  (): TransactionRefundRequestedEventFragment => ({
+    action: {
+      amount: 100,
```

Comment:
lets test with non-optimistic amount, maybe even use some japanese or other non-dollar currency + use decimal cents?

(we can do that in the other PR but I think its worth it)